Makefile

```diff
@@ -2,9 +2,10 @@ VERSION ?= $(shell git describe --tags --dirty)
 UNAME := $(shell uname)
 ARCH := $(shell uname -m)
 APP_NAME = embedded-cluster
+KOTS_VERSION = v1.108.2
 ADMIN_CONSOLE_CHART_URL = oci://registry.replicated.com/library
 ADMIN_CONSOLE_CHART_NAME = admin-console
-ADMIN_CONSOLE_CHART_VERSION = 1.108.1
+ADMIN_CONSOLE_CHART_VERSION = $(subst v,,$(KOTS_VERSION))
```

Comment:
you'll need to update the dependency update github action to handle this and not undo your work

Reply:
good catch.

Reply:
fixed.